client/src/user/ForgetPassword.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+
+import Button from "../common/button.component";
+import Label from "../common/label.component";
+import InputError from "../common/input_error.component";
+import { forgetPasswordValidationSchema } from "./user.schema";
+
+const ForgetPassword = () => {
+    const initialValues = {
+        email: "",
+    };
+
+    const onSubmit = (values) => {
+        console.log("values", values);
+    };
+    return (
+        <>
```

Comment:
extra react fragment is not needed in here. you are just adding an extra div to your layer! are you too much noob???